### ISSUE TITLE
Narrator doesn't announce AccessibilityValue changes

### ISSUE THREAD

**Issue by AgneLukoseviciute** [microsoft/react-native-windows#10284]
Changing the value of accessibilityValue is not triggering Narrator to read out the new state.

Similar to: https://github.com/microsoft/react-native-windows/issues/8491, needs calls to  peer.RaisePropertyChangedEvent when value changes.

**Comment by AgneLukoseviciute** [comment]
Needed for: https://github.com/microsoft/react-native-gallery/issues/230

**Comment by AgneLukoseviciute** [comment]
In PR, should go through soon. keeping in .70
